src/styles/_utils.scss

```diff
@@ -0,0 +1,21 @@
+@mixin square-size ($width) {
+  width: $width;
+  height: $width;
+}
+
+@mixin arrow-animation($time) {
+  animation-name: move;
+  animation-duration: $time * 1s;
```

Comment:
`* 1s` rewrite it without `*`

Reply:
I don’t know how else to indicate that these are seconds, because I just pass a number to the mixin